packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -376,6 +377,11 @@ module.exports = {
         minifyURLs: true,
       },
     }),
+    new DynamicCdnWebpackPlugin({
+      only: ['react', 'react-dom'],
```

Comment:
does it work when the app is cached using service worker and the network is offline? IMO this needs to be opt-in if we want to move forward with it

Reply:
this is a decision the maintainers should take. we already discussed the key motivations in https://github.com/facebook/create-react-app/issues/2758 and I am in favor but I understand your concern.

as far as SW and CRA are concerned I know we are moving towards making them opt-in: https://github.com/facebook/create-react-app/issues/2554.

IMO CRA should cater to the 90% use case as we always have eject as an option. thats what its for.

Reply:
yea but if people chose to opt-in for SW, it broke their expectation because the created app is no longer working offline.. I think it should be a choice between cdn capable vs offline capable. Or maybe we can configure `sw-precache` to cache the files from cdn?